### PR TITLE
Remove old formulae and update kvm for beta3

### PR DESCRIPTION
**Pull request by anurse**
Also adds "brew install --devel" support.

/cc @suhasj @Praburaj @lodejard

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@anurse__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla2.msopentech.com.
    </span>

TTYL, MSOTBOT;

**Comment by anurse** [comment]
Guh. I swear I've filled out the form that tells MSOTBOT that I'm an MSFT employee... just taking time to propagate I guess :).

File: kmono.rb [hunk 1]
```diff
@@ -1,83 +0,0 @@
-require "formula"
-
-class Kmono < Formula
-  homepage "http://www.mono-project.com/"
```
Comment (Praburaj) [inline kmono.rb]: :+1:

File: kre.rb [hunk 2]
```diff
@@ -1,22 +0,0 @@
-require "formula"
-
-class Kre < Formula
```
Comment (Praburaj) [inline kre.rb]: :+1:

File: kvm.rb [hunk 3]
```diff
@@ -2,9 +2,14 @@
 
 class Kvm < Formula
   homepage "https://www.github.com/aspnet/Home"
-  version "1.0.0-beta1"
+  version "1.0.0-beta3"
```
Comment (Praburaj) [inline kvm.rb]: Do we need this version if we have ability to say `master` or `dev`.

Comment (Praburaj) [inline kvm.rb]: And if I install `dev` will `1.0.0-beta3` be relevant?

Comment (anurse) [inline kvm.rb]: Yeah, I think that what I will actually do is change the URL to point at a specific tag for the current version (which doesn't yet exist on aspnet/Home) and use "HEAD" to refer to the tip of the 'dev' branch. We'll have to keep updating the formula, but that's a) pretty easy, and b) basically what [others do](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/rbenv.rb) (though they often build release packages in `.tar.gz` form)

Comment (anurse) [inline kvm.rb]: I'll post an update with those changes soon.

File: kvm.rb [hunk 4]
```diff
@@ -2,8 +2,10 @@
 
 class Kvm < Formula
   homepage "https://www.github.com/aspnet/Home"
-  version "1.0.0-beta1"
-  url "https://github.com/aspnet/Home.git", :branch => 'master'
+  version "1.0.0-beta3"
+  url "https://github.com/aspnet/Home.git", :tag => 'v1.0.0-beta3'
+
+  head "https://github.com/aspnet/Home.git", :branch => 'dev'
```
Comment (anurse) [inline kvm.rb]: This more closely follows the pattern of other Formulae. A couple notes:

1. The tag 'v1.0.0-beta3' does not currently exist. We should not merge this change until it is created, which will happen at beta3 final release time
2. When you install via "brew install kvm --HEAD" you get the `dev` branch, and it has a special version number "HEAD" which is treated as the absolute latest version, so the version is irrelevant in that case.

Comment (suhasj) [inline kvm.rb]: can we have long term solution where we don't need to keep updating this every release ?

Comment (anurse) [inline kvm.rb]: That's pretty uncommon in the Homebrew world. Every formula I could find in the [main repository](https://github.com/Homebrew/homebrew/tree/master/Library/Formula) uses explicit version numbers in the URL of the package to download. The `version` field can be auto-detected from the URL, but it doesn't seem to work with our tag format, and even then we have to explicitly specify the tag. So regardless, we're going to have the version embedded at least once in this file.

At least with this change we get two benefits:

1. Users who want to update to the bleeding edge can use "brew install kvm --HEAD"
2. We only have to update the version number for future updates, no other changes are required.

Comment (suhasj) [inline kvm.rb]: ah necessary evil. It looks good then

**Comment by anurse** [comment]
Ok, sounds like we're ready to go. I'll merge this once the v1.0.0-beta3 tag exists on aspnet/Home

**Pull request merged**